ambry-named-mysql/src/integration-test/java/com/github/ambry/named/MySqlNamedBlobDbIntegrationTest.java

```diff
@@ -79,6 +85,7 @@ public MySqlNamedBlobDbIntegrationTest() throws Exception {
   public void testPutGetListDeleteSequence() throws Exception {
     int blobsPerContainer = 5;
 
+    time.setCurrentMilliseconds(System.currentTimeMillis());
```

Comment:
I kind of feel that it's not scalable that people have to update the currentTime each time when we access the DB. Is there any way to avoid this?

Reply:
We only set this in the testing code. In actual service code, we don't need to update the currentTime at all. Here we are updating the MockTime's currentTime to do the time travel to mimic and test out the actual use case.

Reply:
you can create a MockTime with initial value as System.currentTimeMillis().

Reply:
Done.